docs/ReferenceGuides/UnderscoredAttributes.md

```diff
@@ -117,6 +117,27 @@ extension Text {
 }
 ```
 
+## `@_documentation(metadata: ...)`
+
+Adds "documentation metadata" to the symbol. The identifier in the attribute is
+added to the symbol graph in the `"metadata"` field of the symbol. This can be
+used to add an arbitrary grouping or other indicator to symbols for use in
+documentation.
```

Comment:
Are you able to specify multiple `@_documentation(metadata: …)` attributes?

Reply:
I'm not sure; i'll have to check. Offhand, i think the answer is "no" because of checks for duplicate attributes, but i'm not confident about it.